doc/man7/OSSL_PROVIDER-base.pod

```diff
@@ -57,25 +57,93 @@ currently permitted.
 
 The OpenSSL base provider supports these operations and algorithms:
 
+=head2 Random Number Generation
+
+The default provider all includes the "SEED-SRC" algorithm, which has the
```

Comment:
Is this meant to say "The default provider includes", or do we actually name the default provider "all"?

Reply:
It is factually incorrect.  Neither the base nor default providers claim FIPS=yes for this.  The reason being that this is outside the FIPS boundary.  That's why we've got an entropy caveat in place.

Reply:
Is this used to seed the FIPS provider? I assume it isn't because otherwise running with fips=yes would not be possible.

Reply:
> Is this meant to say "The default provider includes", or do we actually name the default provider "all"?

It was supposed to say "also includes". Now written in actual English.

> It is factually incorrect. Neither the base nor default providers claim FIPS=yes for this. The reason being that this is outside the FIPS boundary. That's why we've got an entropy caveat in place.

Fixed.

> Is this used to seed the FIPS provider? I assume it isn't because otherwise running with fips=yes would not be possible.

Yes. There is some special casing here.

https://github.com/openssl/openssl/blob/9890cc42daff5e2d0cad01ac4bf78c391f599a6e/crypto/rand/rand_lib.c#L550-L596

Reply:
Ah, I forgot that Pauli added that `-fips` override.